gremlin-go/driver/client.go

```diff
@@ -71,9 +75,15 @@ func NewClient(url string, configurations ...func(settings *ClientSettings)) (*C
 		KeepAliveInterval:            keepAliveIntervalDefault,
 		WriteDeadline:                writeDeadlineDefault,
 		ConnectionTimeout:            connectionTimeoutDefault,
+		EnableCompression:            false,
 		NewConnectionThreshold:       defaultNewConnectionThreshold,
 		MaximumConcurrentConnections: runtime.NumCPU(),
 		Session:                      "",
+		// ReadBufferSize and WriteBufferSize specify I/O buffer sizes in bytes. If a buffer
+		// size is zero, then a useful default size is used. The I/O buffer sizes
+		// do not limit the size of the messages that can be sent or received.
```

Comment:
or better to use default 4096?

Reply:
From what I read online, it will default if you have 0. I think we can rely on their defaults instead of arbitrating our own